content/docs/command-reference/exp/init.md

```diff
@@ -155,23 +150,15 @@ The easiest route is using interactive mode and answering a few questions:
 
 ```dvc
 $ dvc exp init --interactive
-This command will guide you to set up a train stage in dvc.yaml...
-
 Command to execute: python src/train.py
 
-Enter the paths for dependencies and outputs of the command.
-DVC assumes the following workspace structure:
-├── data
-├── metrics.json
-├── models
-├── params.yaml
-├── plots
-└── src
-
+Enter experiment dependencies.
 Path to a code file/directory [src, n to omit]: src/train.py
 Path to a data file/directory [data, n to omit]: data/features
-Path to a model file/directory [models, n to omit]: models/predict.h5
 Path to a parameters file [params.yaml, n to omit]:
+
+Enter experiment outputs.
+Path to a model file/directory [models, n to omit]: models/predict.h5
 Path to a metrics file [metrics.json, n to omit]:
 Path to a plots file/directory [plots, n to omit]: n
 ...
```

Comment:
Now that the output is much shorter, can we include it all instead of truncating it?

Reply:
`- epochs` should also be removed in the `dvc.yaml` below.

Reply:
Added full output. Removed `params` group from dvc.yaml sample.